src/common.h

```diff
@@ -277,6 +527,40 @@ extern void get_system_HZ(void);
 #endif
 #endif
 
+/**
+ * Read `filename` containing a single number and return it.
+ *
+ * Open `filename`, read the first line, convert it to a number and store it in result.
+ *
+ * @param filename File to read.
+ * @param result Long to store the number in.
+ * @return 0 on success, -1 if could not open file, -2 if could not read from file.
+ */
 extern int read_single_number_file(const char *filename, unsigned long long *result);
 
+/// \todo ktsaou: your help needed
```

Comment:
haha... this is old... you need to fetch the latest.

To understand simple patterns, check this https://github.com/firehol/netdata/issues/1571#issuecomment-272705918

You can run `netdata -W simple-pattern` for more help or to play with it...